### ISSUE TITLE
Add function to get Ostentus firmware version

### ISSUE THREAD

**Issue by cdwilson** [golioth/libostentus#8]
It would be nice if we could read the Ostentus firmware version from the Zephyr side of things to check whether the display is running the latest firmware.

**Comment by szczys** [comment]
Agreed! We have this on the backlog but so far I haven't been able to get Ostentus to send data back to Zephyr over i2c. We're blocked until the low level issues in that repo can be ironed out.
